tchannel-core/src/main/java/com/uber/tchannel/api/TChannel.java

```diff
@@ -354,6 +357,15 @@ public Builder(@NotNull String service) {
             return this;
         }
 
+        public @NotNull Builder setChildLoadControl() {
+            return setChildLoadControl(8, 32);
```

Comment:
how did you come up with 8 and 32 ?

Reply:
There is no rationale behind this, really. I will remove this method